spec/integrations/posts_index_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+describe 'posts index', type: :system do
```

Comment:
- [ ] As stated in the requirements, let's kindly write a unit tests `that asserts you can see a section for pagination if there are more posts than fit on the view.` :+1: